config/locales/diaspora/en.yml

```diff
@@ -1308,6 +1308,40 @@ en:
       email_confirmed: "Email %{email} activated"
       email_not_confirmed: "Email could not be activated. Wrong link?"
 
+  two_factor_auth:
+    title: "Two factor authentication"
+    activated:
+      status: "Two factor authentication activated"
+      change_label: "Deactivate two factor authentication by entering a TOTP token."
+      change_button: "Deactivate"
+    deactivated:
+      status: "Two factor authentication not activated"
+      change_label: "Activate two factor authentication"
+      change_button: "Activate"
+    confirm:
+      title: "Confirm activation"
+      status: "Two factor authentication is not fully activated yet, you need to confirm activation with a TOTP token"
+      scan_title: "Scan the QR code"
+      scan_explanation: "Please scan the QR code with a TOTP capable app (eg FreeOTP (Android, iOS), SailOTP (SailfishOS))."
```

Comment:
I suggest rephrasing this to avoid the double closing bracket at the end:

> "Please scan the QR code with a TOTP capable app, such as FreeOTP (Android, iOS), SailOTP (SailfishOS)."